scripts/generate-json-schema.ts

```
@@ -210,42 +220,65 @@
-		Result.try({
-			try: writeFile(SCHEMA_FILENAME, schemaJson),
-			safe: true,
-		}),
-		Result.inspectError((error) => {
-			logger.error(`Failed to write ${SCHEMA_FILENAME}:`, error);
-			process.exit(1);
-		}),
-		Result.inspect(() => logger.info(`✓ Generated ${SCHEMA_FILENAME}`)),
-	);
+	// Check if existing schema is identical to avoid unnecessary writes
+	const existingRootSchema = await readFile(SCHEMA_FILENAME);
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)